parastell/magnet_coils.py

```diff
@@ -13,6 +13,8 @@
     import_geom_to_cubit,
     export_mesh_cubit,
     mesh_volume_skeleton,
```

Comment:
Can we delete this? And it seems like it won't be used anymore, should we delete the method from `cubit_utils`?

Reply:
Although, I suppose I could remove it now and if I want to restore it, I can just copy-paste from an old commit

Reply:
I'm fine to leave it there for posterity... is it "cleaned up" with respect to the standard meshing parameters that you've used elsewhere?